src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -37,7 +37,7 @@ const selectedChannelKeyTab = atom({
   default: 0
 });
 
-function useChatUtilsHook(): UseChatUtilsHook {
+function useChatUtilsHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatUtilsHook {
   const [chat, setChat] = useRecoilState(chatAtom);
   const {selectedChannel, isLoadingFetchingChannelDetail} = chat;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `useChatUtilsHook` now accepts a parameter `type`. This is a good change as it allows the hook to be more flexible and adaptable to different contexts. However, there's no default value for `type`, which could potentially lead to unexpected behavior if the function is called without arguments. Consider providing a default value.
```diff
- function useChatUtilsHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatUtilsHook {
+ function useChatUtilsHook(type: 'SIGNED' | 'ANONYMOUS' = 'SIGNED'): UseChatUtilsHook {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->